frontend_v2/src/styles/base.scss

```diff
@@ -2154,13 +2154,10 @@ tr {
     zoom: 1 !important;
   }
 
-  .web-container {
```

Comment:
@Alabhya268 why are we removing this css class? Are you sure it is not messing up css style for other pages?

Reply:
@Ram81 , I sure it's not having any negative effects on css style. I probably can find other way around but that seems unecessary. Let me know what you think.